exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/ddl/DatabaseMigrationTests.kt

```diff
@@ -257,11 +261,585 @@ class DatabaseMigrationTests : DatabaseTestsBase() {
                 SchemaUtils.create(testTableWithIndex)
                 assertTrue(testTableWithIndex.exists())
 
-                val statements = MigrationUtils.statementsRequiredForDatabaseMigration(testTableWithoutIndex)
+                val statements = MigrationUtils.statementsRequiredForDatabaseMigration(testTableWithoutIndex, withLogs = false)
                 assertEquals(1, statements.size)
             } finally {
                 SchemaUtils.drop(testTableWithIndex)
             }
         }
     }
+
+    @Test
+    fun testAddAutoIncrementToExistingColumn() {
+        val tableWithoutAutoIncrement = object : IdTable<Long>("test_table") {
+            override val id: Column<EntityID<Long>> = long("id").entityId()
+        }
+        val tableWithAutoIncrement = object : IdTable<Long>("test_table") {
+            override val id: Column<EntityID<Long>> = long("id").autoIncrement().entityId()
+        }
+
+        withTables(excludeSettings = listOf(TestDB.SQLITE), tableWithoutAutoIncrement) { testDb ->
+            assertEquals(0, MigrationUtils.statementsRequiredForDatabaseMigration(tableWithoutAutoIncrement, withLogs = false).size)
+
+            val statements = MigrationUtils.statementsRequiredForDatabaseMigration(tableWithAutoIncrement, withLogs = false)
+            when (testDb) {
+                TestDB.POSTGRESQL, TestDB.POSTGRESQLNG -> {
+                    assertEquals(3, statements.size)
+                    assertEquals("CREATE SEQUENCE IF NOT EXISTS test_table_id_seq START WITH 1 MINVALUE 1 MAXVALUE 9223372036854775807", statements[0])
+                    assertEquals("ALTER TABLE test_table ALTER COLUMN id SET DEFAULT nextval('test_table_id_seq')", statements[1])
+                    assertEquals("ALTER SEQUENCE test_table_id_seq OWNED BY test_table.id", statements[2])
```

Comment:
Most of these strings are hard-coded and duplicated across tests. I'm assuming that the point of all these tests is to confirm that `MigrationUtils` functions correctly detects differences and returns the correct group of DDL statements based on the type of differences. So if the DDL statements are not being tested themselves, could there maybe be some private functions that return these strings, so there's only a single place to adjust in the future if syntax logic changes? I get that it might be difficult for the alter statements, but could `createStatement()` and `dropStatement()` at least be used directly for the sequence CREATE and DROP strings?